treenode.go

```diff
@@ -679,6 +681,9 @@ func (n *TreeNodeInstance) SetConfig(c *GenericConfig) {
 	n.configMut.Lock()
 	defer n.configMut.Unlock()
 	n.config = c
+	for id := range n.sentTo {
+		n.sentTo[id] = false
```

Comment:
Indeed, it should only be called once. So I'd still remove this initialization, and either a) a big warning in the comments to only call this method once, or b) check in `SetConfig` if it has been called more than once, then panic.
I'm more for the a) approach.